runner/prepare_benchmark.py

```diff
@@ -275,27 +293,33 @@ def beeline(query):
     "visitDate STRING,adRevenue DOUBLE,userAgent STRING,countryCode STRING," \
     "languageCode STRING,searchWord STRING,duration INT ) " \
     "ROW FORMAT DELIMITED FIELDS TERMINATED BY \\\",\\\" " \
-    "STORED AS TEXTFILE LOCATION \\\"/user/spark/benchmark/uservisits\\\";")
+    "STORED AS %s LOCATION \\\"/user/spark/benchmark/uservisits\\\";" % file_format)
 
   beeline("DROP TABLE IF EXISTS documents;")
   beeline(
-    "CREATE EXTERNAL TABLE documents (line STRING) STORED AS TEXTFILE " \
-    "LOCATION \\\"/user/spark/benchmark/crawl\\\";")
+    "CREATE EXTERNAL TABLE documents (line STRING) STORED AS %s " \
+    "LOCATION \\\"/user/spark/benchmark/crawl\\\";" % file_format)
 
   if opts.parquet:
-    ssh_spark("/root/spark/sbin/stop-thriftserver.sh")
-
-    print "Sleeping for 30 seconds so the jdbc server can stop"
-    time.sleep(30)
-
-    scp_to(opts.spark_host, opts.spark_identity_file, "root", "parquet/convert_to_parquet.py",
-      "/root/convert_to_parquet.py")
-    ssh_spark("/root/spark/bin/spark-submit --master spark://%s:7077 /root/convert_to_parquet.py" % opts.spark_host)
-
-    ssh_spark("/root/spark/sbin/start-thriftserver.sh --executor-memory %s" % opts.executor_memory)
-
-    print "Sleeping for 30 seconds so the jdbc server can start"
-    time.sleep(30)
+    if (not opts.skip_parquet_conversion):
+      print "Converting benchmark data to parquet"
+      ssh_spark("/root/spark/sbin/stop-thriftserver.sh")
+      print "Sleeping for 30 seconds so the jdbc server can stop"
+      time.sleep(30)
+
+      scp_to(
+        opts.spark_host, opts.spark_identity_file,
+        "root",
+        "parquet/convert_to_parquet.py",
+        "/root/convert_to_parquet.py")
+      ssh_spark(
+        "/root/spark/bin/spark-submit --master spark://%s:7077 /root/convert_to_parquet.py" %
+        opts.spark_host)
+      ssh_spark(
+        "/root/spark/sbin/start-thriftserver.sh --executor-memory %s" % opts.executor_memory)
+
```

Comment:
nit: can you move this newline to before the command to start the thriftserver, so that all of the thriftserver commands are grouped?